resource/wrapper/retryresource/retry_resource.go

```diff
@@ -0,0 +1,123 @@
+package retryresource
+
+import (
+	"context"
+
+	"github.com/giantswarm/backoff"
+	"github.com/giantswarm/microerror"
+	"github.com/giantswarm/micrologger"
+
+	"github.com/giantswarm/operatorkit/resource"
+	"github.com/giantswarm/operatorkit/resource/crud"
+	"github.com/giantswarm/operatorkit/resource/wrapper/internal"
+)
+
+type Config struct {
+	BackOff  backoff.Interface
```

Comment:
Is backoff reusable? I thought it holds some state.

Reply:
Uncertain, but this did not change. The retry mechanism always only used the backoff instance. The backoff factory was only configured for the wrapper and this did all not change.